linkis-spring-cloud-services/linkis-service-gateway/linkis-gateway-core/src/main/scala/com/webank/wedatasphere/linkis/gateway/config/GatewayConfiguration.scala

```diff
@@ -37,6 +37,7 @@ object GatewayConfiguration {
 
   val ADMIN_USER = CommonVars("wds.linkis.admin.user", "hadoop")
 
+  val ADMIN_PASSWORD = CommonVars("wds.linkis.admin.password", "9br01wloKk9cKkuX")
```

Comment:
It is recommended to modify the default value to the username, set to: ADMIN_USER.getValue